server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMessages.feature

```diff
@@ -285,3 +285,10 @@ Feature: GetMessages method
     And the preview of the message is "àààà éééé èèèè"
     And the textBody of the message is "àààà\r\n\r\néééé\r\n\r\nèèèè\r\n"
     And the htmlBody of the message is "<html>\r\n  <p>àààà</p>\r\n  <p>éééé</p>\r\n  <p>èèèè</p>\r\n</html>\r\n"
+
+  Scenario: Preview should be normalized in case of long and complicated HTML content
+    Given the user has a message "m1" in "INBOX" mailbox with long and complicated HTML content
+    When the user ask for messages "m1"
+    Then no error is returned
+    And the list should contain 1 message
+    And the preview of the message is present and normalized
```

Comment:
the problem is : the steps of your test should help understand what the title means. In this case, anybody reading the test case should be able to learn what "normalized" means.
What about : "the preview should not contain consecutive spaces or blank characters" and "the preview should not be empty"